### ISSUE TITLE
License and Dependency Tracking

### ISSUE THREAD

**Issue by dragotin** [cs3org/wopiserver#71]
It would be great if the CI of the Wopiserver would include any mechanism to track all the modules that are used by the code (python dependencies) and which license they have. 

For productising the code that is a requirement.

**Comment by glpatcern** [comment]
Good point. At the same time the list of dependencies is rather static, and in first approximation one can just target the current list, which includes:
* flask
* pyOpenSSL
* PyJWT
* requests
* more_itertools
* prometheus-flask-exporter
* waitress
